portability-worker/src/main/java/org/dataportabilityproject/worker/ConfigUtils.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2018 The Data Transfer Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.dataportabilityproject.worker;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Map;
+
+final class ConfigUtils {
+  static Map<String, Object> parse(InputStream in) throws IOException {
+    ObjectMapper mapper = new ObjectMapper(new YAMLFactory());
+    return mapper.readValue(in, Map.class);
+  }
+}
```

Comment:
nit: new line at EOF

Reply:
this is weird.. I have the setting configured to add a new line in Intellij. But then I have to manually add *another* newline at the bottom for it to add line 31 here. In Intellij, it shows the cursor on line 32.. has that happened to you?

Reply:
Thats odd. I haven't run into this and tried to reproduce with a new class but it looks like its working fine.

Reply:
yeah, weird. thanks for trying it out.